src/Components/test/testassets/TestContentPackage/ComponentFromPackage.razor

```diff
@@ -1,11 +1,30 @@
-﻿<div class="special-style">
+﻿@using Microsoft.JSInterop
+@inject IJSRuntime JS
+
+<div>
+    The behavior for this button was imported from a JS module shipping with the component.
+    <button @onclick="TriggerPrompt">Trigger promp</button>
+</div>
+
+<div class="special-style">
     This component, including the CSS and image required to produce its
     elegant styling, is in an external NuGet package.
     <button @onclick="ChangeLabel">@buttonLabel </button>
 </div>
 
 @code {
     string buttonLabel = "Click me";
+    private IJSObjectReference _module = null;
+
+    protected async override Task OnInitializedAsync()
+    {
+        _module = await JS.InvokeAsync<IJSObjectReference>("import", "./_content/TestContentPackage/ComponentFromPackage.razor.js");
+    }
+
+    async Task TriggerPrompt()
+    {
+        await _module.InvokeVoidAsync("showPrompt", "Hello from module");
```

Comment:
This doesn't seem to be used by the E2E test code in `JsInitializersTest`, unless I'm missing something. Do you think it would be reasonable to include it in the E2E test coverage? Or if not, should it be removed?

Reply:
Changed it and included a test to validate it